src/Message/Message.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import styles from './Message.module.scss'
+import Icon from '../Icon'
+
+const Message = ({
+  wrapperClassName,
+  className,
+  type,
+  children,
+  icon,
+  iconClassName,
+  ...props,
+}) => (
+  <div
+    className={cx({
+      [`${wrapperClassName} ${styles.wrapper} ${styles[type]}`]: true,
+    })}
+    {...props}
+  >
+    {icon && (
+      <div className={styles.iconWrapper}>
+        <Icon
+          type={icon}
+          className={cx({
+            [styles.icon]: true,
+            [styles[`icon-${type}`]]: type,
```

Comment:
No need for a complicated class like this. By using scss modules, the same thing could be achieved by using 
```scss
// example
.warn {
  // some styles
  .icon {
    // some specific styles for elements 
  }
}
```

Reply:
Done.